ext/openssl/openssl.c

```diff
@@ -6584,7 +6580,7 @@ PHP_FUNCTION(openssl_seal)
 	const EVP_CIPHER *cipher;
 	EVP_CIPHER_CTX *ctx;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "szza|sz", &data, &data_len,
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "szza|s!z", &data, &data_len,
```

Comment:
Default `"RC4"` instead?

Reply:
Same for other $method arguments.

Reply:
We should actually change this to be required. Using RC4 by default (which is currently done) is not good as RC4 is unsecure...

Reply:
Also a good point. I assume that changing the default is not possible, as it's not stored in the output? (I.e., one has to manually pass the same method to openssl_seal and openssl_open?)

Reply:
That's right. The output is just a ciphertext so one has to manually pass the method.

Reply:
Done in #6093, so this needs a rebase now.

Reply:
Thanks! I'll do it in the evening!